great_expectations/expectations/metrics/multicolumn_map_metrics/multicolumn_sum_equal.py

```diff
@@ -10,9 +14,20 @@ class MulticolumnSumEqual(MulticolumnMapMetricProvider):
     condition_value_keys = ("sum_total",)
     condition_domain_keys = ("batch_id", "table", "column_list")
 
-    # TODO: <Alex>ALEX -- temporarily only a Pandas implementation is provided (others to follow).</Alex>
+    # TODO: <Alex>ALEX -- temporarily only Pandas and SQLAlchemy implementations are provided (others to follow).</Alex>
     @multicolumn_condition_partial(engine=PandasExecutionEngine)
     def _pandas(cls, column_list, **kwargs):
         sum_total = kwargs.get("sum_total")
         row_wise_cond = column_list.sum(axis=1) == sum_total
         return row_wise_cond
+
+    @multicolumn_condition_partial(engine=SqlAlchemyExecutionEngine)
+    def _sqlalchemy(cls, column_clause, **kwargs):
```

Comment:
@NathanFarmer Let us use `column_list` in the method signature (rather than column_clause) -- to be consistent with the style.